core/CMakeLists.txt

```diff
@@ -32,6 +32,21 @@ elseif(UNIX) # LINUX or APPLE
     set(dynamic_library_c_file ./adapters/dynamic_library_linux.c ./adapters/gb_library_linux.c )
 endif()
 
+# If using the appropriate build script based on OS that it is used
+if (${enable_native_remote_modules} OR ${enable_java_remote_modules})
```

Comment:
What is the decision behind building libuv versus installing it? Should we check to see if the package is installed on the system before building it?